src/validation/validationSchema.ts

```diff
@@ -206,9 +206,15 @@ export default {
         States: true
     },
     // State machines nested within Map and Parallel states
-    NestedRoot: {
+    NestedParallelRoot: {
         Comment: true,
         StartAt: true,
         States: true
+    },
+    NestedMapRoot: {
+        Comment: true,
+        StartAt: true,
+        States: true,
+        ProcessorConfig: true
```

Comment:
`ProcessorConfig` isn't a valid property for `Iterator` which is another type of nested Map root. However, this shouldn't be an issue since the JSONSchema should invalidate this

Reply:
Disregard — `Iterator` does allow `ProcessorConfig`